### DISCUSSION TITLE
Error:self signed certificate in certificate chain

### DISCUSSION THREAD

**Discussion by Bajibabup** [webdriverio/codemod#85] [General]
Hi,
I started using selenium-standalone-service(Version 8.3.2) for WebdriverIO. Its been installed in node modules successfully.
However, while executing the tests I am getting below error.
I have verified the folder "selenium-standalone\.selenium", found that all web-driver folders empty, selenium-server folder is also empty. I manually placed the concerned webdriver Zip folders and exe files in the concerned folder. But, still same issue. I think my company proxy settings might me blocking it. Please can you help to fix this issue.

Error Info:
"getDownloadStream". Could not download https://selenium-release.storage.googleapis.com/3.150/IEDriverServer_x64_3.150.1.zip
See more details below:
self signed certificate in certificate chain
C:\UIFraemwork\node_modules\selenium-standalone\lib\install.js:292
          throw new Error('Could not download ' + downloadUrl);

**Comment by christian-bromann** [comment]
@Bajibabup this seems like a proxy issue, are you behind a corporate proxy?

**Comment by Bajibabup** [comment]
YesSent from my iPhoneOn Feb 15, 2023, at 1:30 PM, Christian Bromann ***@***.***> wrote:﻿
@Bajibabup this seems like a proxy issue, are you behind a corporate proxy?

**Reply by Bajibabup** [reply]
I am trying to find out if there is any workaround to overcome this issue, by pass the proxy.

**Reply by christian-bromann** [reply]
See project docs: https://webdriver.io/docs/proxy

**Reply by Bajibabup** [reply]
Thanks Chris!
I provided following in the wdio.config file, process.env.NODE_TLS_REJECT_UNAUTHORIZED = "0".
Now, I could able to pass proxy error and all drivers, selenium server has downloaded. Firefox browser is being opened, but no tests are executed in that browser. Based on   connectionRetryCount setting, it tried 3 times, finally tests were executed in Chrome.
I know, its a separate issue. I can start a separate discussion if needed. Do I need provide any path for Geko driver, or any other issue?

**Reply by christian-bromann** [reply]
I am not sure if I can help you given you have a custom set up. I can only refer to the project docs.

**Reply by bajipogadadanda** [reply]
Thx!
